### DISCUSSION TITLE
still hope to upload bin files to the slingbox?

### DISCUSSION THREAD

**Discussion by regmar505** [GerryDazoo/Slinger#569] [General]
Gerry,  in November 2022 you wrote:

> Without the current password you have no choice but to factory reset the box to reset the admin password to "admin". You should be able to stream from the box but the integrated IR blaster will not work (yet, we're working on it, don't give up hope).

Just curious to know whether there is still hope to upload bin files to the slingbox. Thanks.

**Comment by mike-uk** [comment]
Which box do you have I just resetted my Pro-HD?

**Reply by regmar505** [reply]
Slingbox M2. I just moved it to a new location and the bin file for the remote control previously uploaded to the Slingbox needs to be replaced. It is not possible to use version SBD 1.5 to reset the M2 as you did with the Pro-HD.

If the worst comes to the worst, does anyone have success with an IR blaster where codes could be sent via internet to the cable box?

**Reply by f520k510** [reply]
I use broadlink rm4 mini.  It allows me to send ir signal through internet.
The price is less than usd 10 for rm4 mini.  It uses smartphone or tablet app to remote controll the devices.

**Reply by flipper2222222** [reply]
I use a harmony hub,  It allows me to send ir signal through internet.

**Reply by regmar505** [reply]
Thanks to both of you for your suggestions. May I ask how the interface looks like on the phone for the rm4 mini? Do you get a replica of a physical remote control?

**Reply by mike-uk** [reply]
I just saw this Tuya s18 even cheaper. 

https://a.aliexpress.com/_EyBd4LD

**Reply by remgr** [reply]
Yes, I bought mine from Aliexpress as well, but you have to be patient with delivery times.  The Tuya is a Moes.  I just sent the amazon link to show you the product.  A couple of things to note, the one listed on Ali does not come with a power adapter.  I also found that on some of them, you have to hit the reset button before you can connect to it.  Finally, it only works on 2.4 mhz. 

It has hundreds of remote codes available, and once setup, it works flawlessly.

**Reply by mike-uk** [reply]
I may give the tuya one a try. I just order an ir sensor for a remote learning circuit and that cost almost as much as that alone. It uses usb to power so any adaptor will work. I wonder if the pro-hd usb port is enough to power it.

**Comment by remgr** [comment]
This is what I use.  It works perfectly, I have setup two other family members with these for their slingboxes:
https://www.amazon.com/MOES-Controller-Infrared-Universal-Compatible/dp/B07QH1X7PX

**Comment by remgr** [comment]
Here is the power adapter that comes with the Moes.  At least you can
compare the power available on you Slingbox USB.
